Assets/Scripts/Game/UserInterfaceWindows/CreateCharCustomClass.cs

```diff
@@ -394,6 +401,15 @@ void ReputationButton_OnMouseClick(BaseScreenComponent sender, Vector2 position)
             uiManager.PushWindow(createCharReputationWindow);
         }
 
+        protected virtual void ResetButton_OnKeyboardEvent(BaseScreenComponent sender, Event keyboardEvent)
+        {
+            if (keyboardEvent.type == EventType.KeyDown)
+            {
+                DaggerfallUI.Instance.PlayOneShot(SoundClips.ButtonClick);
+                statsRollout.BonusPool = 0;
```

Comment:
It seems to be like this should also reset all stats to 50, rather than leave the current values. As it is, it's more of a cheat code to get all stats to 75 for free.

![image](https://github.com/user-attachments/assets/b46e8a5c-0ab8-490e-9674-6448ab71d57a)
![image](https://github.com/user-attachments/assets/c1c92781-e1e4-4c11-89b9-ca41742c10c8)

Reply:
No, it's not a "reset" shortcut.
It operates exactly like in classic, this allows to create unbalanced classes, for example Grimith used it (by running character creation under classic then switching to DFU), in conjunction with LevelUp Adjuster mod, to create a character with very low starting abilities.
Daggerfall is a single player game, while balancing is the standard rule for creating characters, it doesn't need to be a hard rule.